### PR TITLE
Update copyright header

### LINKED ISSUE #1488: Update copyright header on all files
New year <=> licence headers to update on each file (?)

As it's the update time, is the  
```
* @version $Id$
```
 still used ?
On releases, the tag appears not to be computed 

And last question : would it be added a mention like "and all projects contributors" on the header ? (sens ? legal ?)

Regards

### PR DESCRIPTION
**Pull request by AnaelMobilia**
Update copyright header & script tools/modify_header.pl

Replace previous PR due to too many merge conflicts to resolve

Close #1488

### CODE REVIEW

**Comment by trasher** [comment]
Hello, a few comments:
- on commit 3a8e3f8a4192dff08a5aa8d76f35147c5ef76fd6; I disagree, license must be set on those files, they're part of the project;
- on commit 6953ef3153121f3c4d08e164c9ca6d4f27be6aed would not it be better to add a section in the developer docs for that? Also, some scripts may be removed (at last, all svn related files)

**Comment by AnaelMobilia** [comment]
Thanks for review @trasher .
For your first point, on which filetypes (.htaccess, .sql, .sh, .pl, html, .MD) should be set copyright for you ?
(both for me, except .html)

**Comment by trasher** [comment]
That's a good question... I'd say everywhere :p

Well, on scripts, it must be set, just as the sources themselves.

On displayed files, that could be a size problem; but we do not have `.html` files; and comment added using `<!--- My comments -->` are ignored by the parser apprently.

So I guess it is ok to put them everywhere.

Thoughts?

**Comment by AnaelMobilia** [comment]
The [readme has be moved to docdev](https://github.com/glpi-project/docdev/pull/27)

The modify_headers.pl have been updated to handle correctly .sql, .pl and .sh files.
Copyright has been set on theses files with the script.

The script will now alert the dev if a file haven't copyright (but should have it).

For the .md files, we delete it on release => i think that there isn't interest to copyright it

For .htaccess, @orthagh request on previous PR #1536 to don't have copyright header on .htaccess files => not integrated on the script / PR

Regards

File: tools/modify_headers.pl [hunk 1]
```diff
@@ -1,102 +1,117 @@
...
-				
+            # Excluded directories
+			if ($_ !~ m/.git/i && $_ !~ m/lib/i && $_ !~ m/plugins/i && $_ !~ m/vendor/i){
 				do_dir("$dir/$_");
 			}
 		} else {
 	 		if(!(-l "$dir/$_")){
-				if ((index($_,".php",0)!=-1)||(index($_,".txt",0)!=-1)||(index($_,".css",0)!=-1)){
-					do_file("$dir/$_");
+                # Included filetypes - php, css, js
```
Comment (trasher) [inline tools/modify_headers.pl]: I'm not ablo to read perl code; but should it really be commented?

Comment (orthagh) [inline tools/modify_headers.pl]: +1
Without that part, we don't scan php|css|js files

Comment (orthagh) [inline tools/modify_headers.pl]: also, indent for the real comment seems ko

Comment (AnaelMobilia) [inline tools/modify_headers.pl]: @orthagh Thanks for the catch, it's fixed ! ;)

**Comment by trasher** [comment]
Thanks for the work @AnaelMobilia .

**Pull request merged**